src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorage.cs

```diff
@@ -300,6 +325,30 @@ public void Initialize(Solution solution)
             // Try to bulk populate all the IDs we'll need for strings/projects/documents.
             // Bulk population is much faster than trying to do everything individually.
             BulkPopulateIds(connection, solution, fetchStringTable);
+
+            return;
+
+            static void EnsureTables(SqlConnection connection, Database database)
+            {
+                var dbName = database.GetName();
+                connection.ExecuteCommand(
+$@"create table if not exists {dbName}.{SolutionDataTableName}(
```

Comment:
> {dbName}.{SolutionDataTableName}

This is quoted in the original string. Why is it not quoted here?

Reply:
turns out it's not necessary.  so i felt this reads better.